packages/mask/src/plugins/NextID/components/UnbindConfirm.tsx

```diff
@@ -0,0 +1,74 @@
+import { memo, useState } from 'react'
+import { Box, DialogContent, Button } from '@mui/material'
+import { useWeb3State } from '@masknet/plugin-infra'
+import { useI18N } from '../locales'
+import { makeStyles, MaskDialog } from '@masknet/theme'
+
+const useStyles = makeStyles()((theme) => ({
+    wrapper: {
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        flexDirection: 'column',
+        padding: '24px !important',
+    },
+    title: {
+        fontSize: '16px',
+        fontWeight: 500,
+        marginBottom: '24px',
+    },
+    content: {
+        marginBottom: '36px',
+        color: theme.palette.grey[700],
+    },
+    confirmButton: {
+        width: '100%',
+        marginBottom: '16px',
+        backgroundColor: 'red',
+        color: theme.palette.common.white,
+    },
+    cancelButton: {
+        width: '100%',
+        marginBottom: '16px',
+    },
+}))
+
+interface UnbindConfirmProps {
+    unbindAddress: string
+    onClose(): void
+    onConfirm(): void
+}
+
+export enum DialogTabs {
+    persona = 0,
+    wallet = 1,
+}
+
+export const UnbindConfirm = memo<UnbindConfirmProps>(({ onClose, unbindAddress, onConfirm }) => {
```

Comment:
```tsx
<UnbindConfirm unbindAddress={unbindAddress} onConfirm={() => setSecondDialog(true)} onClose={onClose} />
```

A non-memo function is passed to `onConfirm`, then `memo` here can't not reduce rerendering.